config/routes.rb

```diff
@@ -6,6 +6,7 @@
   end
   get '/admin/dashboard', to: 'admin/dashboard#index'
   resources :users, only: [:new, :create]
+  get '/profile', to: 'users#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.